test/functional/terminal/tui_spec.lua

```diff
@@ -870,26 +870,55 @@ describe('TUI background color', function()
     feed_data('\027]11;rgb:ffff/ffff/ffff\007')
     screen:expect{any='did OptionSet, yay!'}
   end)
+end)
+
+describe('TUI deferred background color', function()
```

Comment:
why add a new describe() group? the setup looks the same.

Reply:
Yes.  Also uses a separate/explicit description.